roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -1269,11 +1269,14 @@ impl TweezerDevice {
             }
             true
         }
+        // Temporary clone: pretending the shift of the qubits in order to understand
```

Comment:
What do you mean by pretending here?

Reply:
The fact that in working on a temporary qubit -> tweezer mapping. The rest of the script will reference the temporary one, so that the original is left untouched. If it turns out that all the operations are valid, then `true` will be returned the calling method will handle the real transformation.